src/Raven.Studio/typescript/viewmodels/database/documents/editDocument.ts

```diff
@@ -582,6 +576,10 @@ class editDocument extends viewModelBase {
         copyToClipboard.copy(this.editedDocId(), "Document ID has been copied to clipboard");
     }
 
+    copyChangeVectorToClipboard() {
+        copyToClipboard.copy(_.replace(this.changeVectorFormatted(), new RegExp("<br/> ", "g"), "\n"), "Change Vector has been copied to clipboard");
```

Comment:
can we avoid tying this to html form, instead we can do:
this.changeVector().map(vectorItem => vectorItem.fullFormat).join(' ');

Reply:
done